composer.json

```diff
@@ -4,7 +4,8 @@
     "description": "A Flow package that allows for protecting persistent resources from unauthorized access",
     "license": "MIT",
     "require": {
-        "neos/flow": "^4.1 || ^5.0"
+        "neos/flow": "^6.0",
+        "guzzlehttp/psr7": " ^1.4"
```

Comment:
Thanks! Just one last issue. According to https://semver.mwl.be/#!?package=guzzlehttp%2Fpsr7&version=%5E1.6&minimum-stability=stable `^1.6` would be the last stable version and that's what my current Flow 6.0 installation installs.
Is there a specific reason why you added a 1.*4* dependency?

Reply:
I looked into the composer json of the flow develepment collection and and copied the dependency from there (https://github.com/neos/flow-development-collection/blob/master/composer.json). There `^1.4` is used. Therefore I set the dependency to `^1.4`. But you are totally right. I adjusted the composer json accordingly to `^1.6`.

Reply:
Thank you, Andre!